### PR TITLE
add default HF modal behaviour & fix blank page

### PR DESCRIPTION
**Pull request by AntonNazarenko**
fixed blank page issue
added default HF modal behavior to guide modal

### CODE REVIEW

File: src/components/GridLayoutPage/GridLayoutPage.js [hunk 1]
```diff
@@ -36,12 +35,6 @@ export default class GridLayoutPage extends React.Component {
     layoutDirty: false,
     addLayoutModalOpen: false,
     addComponentModalOpen: false,
-    steps: [
-      {
-        target: '.hfui-button.green',
-        content: 'To customize your layout, you can add components to it',
-      },
-    ],
```
Comment (robertkowalski) [inline src/components/GridLayoutPage/GridLayoutPage.js:44]: where is this section now? did you see https://github.com/bitfinexcom/bfx-hf-ui/commit/91ee33ab8f40ff7b6875c1e649fca11f552b82e5

File: src/pages/MarketData/MarketData.js [hunk 2]
```diff
@@ -1,9 +1,50 @@
...
 import './style.css'
 
 export default class MarketData extends React.PureComponent {
+  static propTypes = propTypes
+
+  state = {
+    steps: [
+      {
+        target: '.hfui-button',
+        content: 'Create new layout',
```
Comment (robertkowalski) [inline src/pages/MarketData/MarketData.js:14]: seems these are old steps / texts? see deleted text above

**Review by robertkowalski — CHANGES_REQUESTED** [review]
would other pages with custom components to add also suffer from this bug? i think there is another page with custom layouts is this affected, too?

File: src/components/GridLayoutPage/GridLayoutPage.js (hunk 1 above)
Comment (AntonNazarenko) [reply src/components/GridLayoutPage/GridLayoutPage.js:44]: @robertkowalski I've moved this section to the Market Data page. Didn't see these changes, will update steps content now

**Comment by AntonNazarenko** [comment]
> would other pages with custom components to add also suffer from this bug? i think there is another page with custom layouts is this affected, too?
@robertkowalski 
I've refactored the code, now everything should be fine

**Comment by AntonNazarenko** [comment]
Снимок экрана 2020-09-29 в 18 22 50
now the guide on the market page contains only one step. Is it correct?  @robertkowalski

**Comment by robertkowalski** [comment]
@AntonNazarenko yes, the other elements should be self explanatory to the user. saving and choosing a template are default behaviours we see in all apps / websites these days, if we have to explain that, we have a problem

**Review by robertkowalski — APPROVED** [review]

**Pull request merged**